<launch>
    <arg name="project" default="lio_sam_6axis"/>

    <include file="$(find lio_sam_6axis)/launch/include/module_loam.launch"/>

    <!--- Run Navsat -->
    <node pkg="lio_sam_6axis" type="lio_sam_6axis_gpsOdometry" name="lio_sam_6axis_gpsOdometry" output="log"/>

    <!--- Run Rviz-->
    <node pkg="rviz" type="rviz" name="$(arg project)_rviz"
          args="-d $(find lio_sam_6axis)/launch/include/config/vlp.rviz"/>

</launch>
